bin/wasabi.sh

```diff
@@ -265,22 +266,17 @@ package() {
 
   ./bin/fpm.sh -n ${name} -v ${version} -p ${profile}
 
+# FIXME: don't rebuild, cp dist/* target/*
```

Comment:
i would like to defer fixing this one at this time ... it requires time to slow down and do it right. we always have subsequent patches, thus the FIXME.